src/main/java/com/adobe/target/edge/client/utils/TargetConstants.java

```diff
@@ -15,5 +15,5 @@
 
   public static final String COOKIE_NAME = "mbox";
   public static final String CLUSTER_COOKIE_NAME = "mboxEdgeCluster";
```

Comment:
Could we move this to a separate PR. We need an automated way to generate and include these details here, either via class loader resources or something similar.

Reply:
As discussed, we should remove this version from code and just read it directly from gradle.properties, which should be copied to src/main/resources during build so it gets packaged in the JAR.
I'll create a separate task for this